test/augmentation/test_container.py

```diff
@@ -8,6 +9,30 @@
 from kornia.geometry.transform import bbox_to_mask
 
 
+def reproducibility_test(input, seq):
```

Comment:
if the test name does not follow `test_*` then pytest won't collect it

Reply:
It is alright. It is general function for performing reproducibility test. It will be called afterwards.